src/AryUtil.chpl

```diff
@@ -9,6 +9,9 @@ module AryUtil
     use MultiTypeSymEntry;
     use ServerErrors;
     use BitOps;
+    use OS.POSIX;
+
+    use ArkoudaPOSIXCompat;
```

Comment:
Do we need to import both of these? If there is a compat module, I feel like normally we just import the compat module.

Reply:
I don't think both are needed. I added `OS.POSIX` to prepare for whenever the compatibility module is removed, but I can remove it for now to avoid ambiguity.